lib/api/core.dart

```diff
@@ -198,6 +199,21 @@ Map<String, String> authHeader({required String email, required String apiKey})
   };
 }
 
+Map<String, String> userAgentHeader() {
+  return {
+    // Putting "ZulipMobile" in the string helps users on pre-8 servers:
+    //   - When a user sends a message from the app,
+    //     the message is read instead of unread, avoiding #440.
+    //     (With servers 8+, this is accomplished
+    //     with a new param in the send-message API; see #456.)
+    //   - Login notifications say something specific instead of
+    //     "an unknown browser on an unknown operating system", avoiding #460.
```

Comment:
Does this latter part work if we say "ZulipFlutter" instead of "ZulipMobile"? (If not, then we might need further server changes but we'll want to figure those out.)

Reply:
Yes, I just tested manually by changing the string from 'ZulipMobile (actually ZulipFlutter)' to just 'ZulipFlutter' and seeing what the login-notification email said, by visiting `/emails` on the dev server. It said:

> Device: Zulip on an unknown operating system.

, the same as it does with 'ZulipMobile (actually ZulipFlutter)' (https://github.com/zulip/zulip-flutter/pull/471#issuecomment-1874756255).

Reply:
Er, and I've just tested this on a dev server at 7.x (zulip/zulip@c4a826461b5d8534ad4653a289623ec7f5c4e70f), and got the same result. I realized that my original test here must have been done on an 8+ dev server.